app/views/posts/index.html.erb

```diff
@@ -24,58 +24,61 @@
       <% end %>
     </div>
   </div>
-  <div class="col-md-12">
-    <%= grid(@post_grid) do |g|
-      g.column name: "Titel", attribute: "title", filter: false do |post|
-        link_to(post.title, %(##{dom_id(post)}), class: "", data: {toggle: "collapse"})
-      end
-      g.column name: "Gräns", attribute: "limit", filter: false do |post|
-        (post.limit) && (post.limit > 0) ? post.limit : "Ingen"
-      end
+  <% if @post_grid.present? && @posts.present? && @posts.count > 0 %>
+    <div class="col-md-12">
+      <%= grid(@post_grid) do |g|
+        g.column name: "Titel", attribute: "title", filter: false do |post|
```

Comment:
We prefer single-quotes for constant strings. Use double-quotes only if you need string interpolation.